src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -141,29 +154,31 @@ public IndexDatabase() throws IOException {
      */
     public IndexDatabase(Project project) throws IOException {
         this.project = project;
-        lockfact = SimpleFSLockFactory.INSTANCE;
+        lockfact = NoLockFactory.INSTANCE;
         initialize();
     }
 
     /**
      * Update the index database for all of the projects. Print progress to
      * standard out.
      *
-     * @param executor An executor to run the job
+     * @param parallelizer a defined instance
```

Comment:
this maybe a little bit more descriptive

Reply:
My reasoning behind this wording is that I think it's more useful to know whether an argument is required or not — hence "a defined instance" or "an optional instance" — rather than recapitulating the description of the class. If the method specifically targeted an argument, I might write e.g., "a defined instance to write" or "a defined instance to append". If the method did something very unintuitive, I would comment.

Reply:
ok that didn't seem obvious to me